src/App.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
+import TodoListItem from './TodoListItem';
+// import Search from './TodoListItem';
```

Comment:
we can remove the unnecessary comments , I don't see this Search component is part of this PR

Reply:
yes it was not. I was practicing. the reason I keep them as comment, is to remember the previous steps and why we applied changes. If there is issue on keeping them, please let me know to remove them. Thanks